compose-macros/src/lib.rs

```diff
@@ -78,31 +75,70 @@ macro_rules! compose_extrinsic_offline {
 /// # Arguments
 ///
 /// * 'api' - This instance of API. If the *signer* field is not set, an unsigned extrinsic will be generated.
+/// * 'nonce' - signer's account nonce: u32
 /// * 'module' - Module name as &str for which the call is composed.
 /// * 'call' - Call name as &str
 /// * 'args' - Optional sequence of arguments of the call. They are not checked against the metadata.
 /// As of now the user needs to check himself that the correct arguments are supplied.
 #[macro_export]
-macro_rules! compose_extrinsic {
+macro_rules! compose_extrinsic_with_nonce {
 	($api: expr,
+	$nonce: expr,
 	$module: expr,
 	$call: expr
 	$(, $args: expr) *) => {
 		{
-            use $crate::log::debug;
+			use $crate::log::debug;
```

Comment:
Is that indent correct?

Reply:
No, I fixed it now. Seems `cargo fmt` works differently in macros. Interesting